### DISCUSSION TITLE
Mirth Connect Installer Not Working - Java Required

### DISCUSSION THREAD

**Discussion by Blackglade** [nextgenhealthcare/connect#6073] [Frequently Asked Questions] [answered]
I'm on an m1 MacOs Ventura (13.3.1) and I installed the latest version of Java but can't seem to get the installed to run. Any idea what is happening. 

It says I need at least Java 1.8 and just exits.

<img width="278" alt="Screenshot" src="https://github.com/nextgenhealthcare/connect/assets/6363077/e072ddae-2acb-4fb2-80dd-4966298c819f">

**Comment by jonbartels** [comment, marked as answer]
Are you trying to install the MC Server or the MC Launcher?

If you are trying to run the server you should instead use:
* The tarball download, just unpack it and run `./mcserver`
* The docker container https://github.com/nextgenhealthcare/connect-docker

Both of those are easier, safer, and less hassle than the installer for the Server.

If you are running the Launcher.. check the install options, is there one that allows you to add Java 8? This will be a local install only for the launcher and will not affect the rest of your mac.

**Reply by Blackglade** [reply]
Thanks! Ended up using the docker container. It was much easier!
